### DISCUSSION TITLE
Why v1.1.7 won't connect to wifi.

### DISCUSSION THREAD

**Discussion by Vslav** [softwarecrash/Solar2MQTT-ESP8266#133] [Q&A]
Thanks for the project, v1.1.6 works great.
But v1.1.7 won't connect to my wifi router. Judging by the router logs, it doesn't even ask. From what I understand there is something wrong with my Wemos D1 Mini module. Since I was able to write the original firmware into it only with the old CH431 drivers.
Am I thinking in the right direction? Has anyone had similar problems?

**Comment by softwarecrash** [comment]
with 1.1.7 the minimum signal quallity was incerased for better stability.
nothing related to ch341 drivers

**Comment by Vslav** [comment]
With my module, on the contrary, the connection does not occur, not to mention its stability. I don't understand if it is because of my module or because of my router. But version 1.1.6 works very well and stably, except for one indicator - the battery temperature, which for some reason is taken from the temperature of the inverter radiator. But this is most likely due to the connection of the inverter with the battery using Pylontech RS485 protocol, which is incorrectly interpreted by the my PowMR inverter.

**Reply by softwarecrash** [reply]
Whats your Setup? Router and more?

**Reply by softwarecrash** [reply]
you can check it, there is no wifi releated changes
https://github.com/softwarecrash/Solar2MQTT/compare/V1.1.6...V1.1.7

**Reply by softwarecrash** [reply]
@Vslav try the pre release, i have found some minimal wlan changes

**Reply by Vslav** [reply]
thanks, i'll try it later, not now. Where I'm at right now, other things are more important right now... sorry

**Reply by tkachenkogenady** [reply]
There were problems with connecting to the router in version V1.1.8
In version V1.2.0-Pre4, everything is fine.

**Reply by Vslav** [reply]
Also in version V1.2.0-Pre everything is fine! 1.7 and 1.8 do not connect. I have now tried both versions. Thank you for your creativity!

**Reply by softwarecrash** [reply]
Thanks, i will Adapt it in the next release

**Reply by Vslav** [reply]
Sorry for being late with the version test... not everything is so good here where we are.

**Reply by softwarecrash** [reply]
No Problem,
take care of yourself

**Reply by NickChayka** [reply]
The same with 2.0.0-Pre 6 version, won't connect to local WiFi. But V1.2.0-Pre4, everything is fine, working as expected.
